docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,111 @@
+---
+description: The Ibexa Cloud product guide provides a full description of its features as well as the benefits it brings to the client.
+---
+
+# [[= product_name_cloud =]] product guide
+
+## What is [[= product_name_cloud =]]
+
+[[= product_name_cloud =]] is a cloud hosting platform that enables you to host your application in the cloud by using the [Platform.sh](https://platform.sh/) service.
+It also establishes the framework for the potential growth and improves project delivery.
+As a diverse Platform as a Service (PaaS), it's intended to let you to concentrate on the important things.
+
+[[= product_name_cloud =]] is a part of [[= product_name =]] - a software that is designed to provide your business with all the features, functionality, and support you need to transform your business for the digital age.
+
+![Ibexa Cloud - part of Ibexa DXP](ibexa_cloud_dxp.png)
+
+## Availability
+
+[[= product_name_cloud =]] is available in all [[= product_name =]] editions.
+
+## How does [[= product_name_cloud =]] work
+
+Ibexa's Cloud Hosting Platform is designed to streamline the development and testing processes, allowing you to deliver new features much faster.
+It ensures that developers spend less time managing databases, queues, search engines, and operating systems and more time to important development activities.
+It shifts the focus away from fundamental operations and toward improving your digital services with additional features, and functionality.
+
+![How does Ibexa Cloud work](ibexa_cloud.png)
+
+## Create account
+
+If you want to use [[= product_name_cloud =]], you need to follow [installation process](install_on_ibexa_cloud.md).
+
+You also need to have an account to log in to https://console.ibexa.cloud.
+
+![Ibexa Cloud - welcome page](ibexa_cloud_login.png)
```

Comment:
This section could suggest to the reader that Ibexa Cloud is free. "Just create an account and start working with Ibexa Cloud." There is probably some limitation to warn about.

I remember having to change things in the Ibexa Cloud training because even using the Platform.sh 30 days trial on Ibexa Cloud became impossible, and the trainee now need to use an existing P.sh plan from his/her company or final client.

Reply:
"_You also need to have an account_" - I didn't use _"create"_ here to don't make this confusion.
Anyway, I can try to add some sentence (like in Ibexa Engage) about the process of requesting the access. Will do, when I will confirm how does it work.

Reply:
"Create" was used in the section title. I see you renamed and reworked the whole section. Good job.